docs/requirements-documentation.txt

```diff
@@ -6,4 +6,5 @@ sphinx-rtd-theme==1.0.0
 torch==1.10.2
 nbconvert==7.0.0
 ipython==8.0.1
-scikit-image==0.19.3
\ No newline at end of file
+scikit-image==0.19.3
+ipywidgets==8.0.4
```

Comment:
Also, you would notice, that there is a requirements.txt file. Those are requirements we have for a dev setup, and running tests also. The reason I'd do it in setup.py, because ipywidgets is a non dev dependency in setup.py, and we should pin the package version there to keep things clear.